service_quotas/ec2_limits.go

```diff
@@ -304,12 +304,12 @@ func (c *AvailableIpsPerSubnetUsageCheck) Usage() ([]QuotaUsage, error) {
 						return true
 					}
 					maxNumOfIPs := math.Pow(2, 32-float64(blockedBits))
-					usage := (maxNumOfIPs - float64(*subnet.AvailableIpAddressCount)) / maxNumOfIPs
+					usage := float64(maxNumOfIPs - float64(*subnet.AvailableIpAddressCount))
```

Comment:
were we just reporting an incorrect usage before? woops 😬

Reply:
Yup, they were reporting the usage % and everything else was reporting number used.